internal/client.go

```diff
@@ -405,6 +405,20 @@ type (
 		TLS                *tls.Config
 		DisableHealthCheck bool
 		HealthCheckTimeout time.Duration
+		// Enables keep alive ping from client to the server, which can help drop abruptly closed connections faster.
+		EnableKeepAliveCheck bool
+		// After a duration of this time if the client doesn't see any activity it
+		// pings the server to see if the transport is still alive.
+		// If set below 10s, a minimum value of 10s will be used instead.
```

Comment:
Just checking that 10s min is enforced, doesn't show up obviously in this diff.

Reply:
It's a gRPC default.

Reply:
It is enforced by gRCP itself.